devops/ansible/site.retry

```diff
@@ -0,0 +1 @@
+localhost
```

Comment:
This shouldn't be commit, we should add it to the .gitignore ...

Reply:
Removed